<ui version="4.0" >
 <class>search_engine</class>
 <widget class="QWidget" name="search_engine" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="search_lbl" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Search Pattern:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_pattern" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearPatternButton" >
       <property name="maximumSize" >
        <size>
         <width>34</width>
         <height>29</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >
		<normaloff>:/Icons/oxygen/edit_clear.png</normaloff>:/Icons/oxygen/edit_clear.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>30</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_button" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>29</height>
        </size>
       </property>
       <property name="text" >
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_search_button" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>29</height>
        </size>
       </property>
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enginesButton" >
       <property name="text" >
        <string>Search engines...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="status_lbl" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="search_status" >
       <property name="minimumSize" >
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Stopped</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>188</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" />
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="download_button" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>601</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>search_pattern</sender>
   <signal>returnPressed()</signal>
   <receiver>search_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>685</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
